### ISSUE TITLE
Point cloud completion for noisy partial sensor data

### ISSUE THREAD

**Issue by gabrielazarzar** [wentaoyuan/pcn#36]
Hi! I would like to use your code to complete noisy partial sensor data. In this case, how should I preprocess my point cloud before feeding it to the pre-trained model you provide? E.g. should the point cloud be zero-centered and/or normalized/scaled in some particular way? Thanks!

**Comment by wentaoyuan** [comment]
The data should be segmented (i.e. containing single object), zero-centered and normalized in the unit cube. If you are using the pretrained model directly without fine-tuning, make sure the test data is similar enough to shapes in ShapeNet.
